src/index.html

```diff
@@ -1,17 +1,132 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
-  </head>
-  <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
-  </body>
+<!DOCTYPE html>
+<link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <form>
+    <fieldset class="group">
+      <legend>Personal information</legend>
+      <div class="inputEl">
+        <label for="Surname">Surname:</label>
+        <input
+        name="Surname"
+        type="text" autocomplete="off"
```

Comment:
- if you have more than 2 attributes - each should be on the new line
- those should have 2 indents if moved to a new line
```suggestion
          name="Surname"
          type="text" 
          autocomplete="off"
```